initdb.py

```diff
@@ -114,8 +107,8 @@ def create_tables(cursor, connection):
             print("Creating table {}: ".format(name), end='')
             # Execute the CREATE xxx in TABLES
             cursor.execute(query)
-        except mysql.connector.Error as mysqlError:
-            raise Exception('There was a problem creating table {}:\n{}'.format(name, mysqlError))
+        except mysql.connector.Error as MySqlError:
+            raise Exception('There was a problem creating table {}:\n{}'.format(name, MySqlError))
         else:
             print("OK")
```

Comment:
This is probably not needed anymore. We can assume OK upon completion without error.